src/core/components/models.jsx

```diff
@@ -45,12 +45,12 @@ export default class Models extends Component {
     const JumpToPath = getComponent("JumpToPath")
 
     return <section className={ showModels ? "models is-open" : "models"}>
-      <h4 onClick={() => layoutActions.show("models", !showModels)}>
-        <span>{isOAS3 ? "Schemas" : "Models" }</span>
+      <div onClick={() => layoutActions.show("models", !showModels)}>
```

Comment:
pinging @smhigley - if you have a moment, could you weigh in on whether these element structure changes are positive, negative, or neutral from an accessibility perspective? No worries if you don't have the time!

Reply:
@shockey happy to comment! I believe this should actually stay a heading, although `<h2>` would be more appropriate.

It's a little hard to set heading levels given swagger UI can be included in a variety of contexts, but without giving devs the option to set heading levels `<h2>` seems like the best choice.

Reply:
@shockey Good point, I was not thinking of accessibility, but it is definitely something we will note going forward...
In the context of _this_ PR - my only concern was the specific padding applied to the heading tag. I'm happy to revert to the original structure and transfer the padding as margins on the inner elements if this makes more sense.

Regarding @smhigley 's comment on appropriate header ranking - an upcoming PR dealing specifically with fonts should address these issues.

Reply:
@Rody-Kirwan sounds good - as subsequent PRs land let's make sure this gets addressed, and to the extent that time permits, optimize for accessibility in general 😄 

Marking as resolved. Thanks again for the assist @smhigley!